packages/terra-menu/src/_MenuUtils.js

```diff
@@ -1,5 +1,11 @@
 import React from 'react';
 
+const stopPropagation = (event) => {
```

Comment:
To be honest, I just wanted to find a place for the comment that is included in that method, explaining why the propagation needs to be stopped there, so that engineers in the future understood why it is there at all:

// stop event propagation in case Menu opened inside the layout component that has its own event handler for that event.
 // currently needed for Menu Button support in `terra-compact-interactive-list`
 
But I agree with you that it adds complexity, I'll remove it an attach directly to onFocus Event. I'll update this comment with the commit once I push it.

Reply:
removed in https://github.com/cerner/terra-framework/pull/2073/commits/670b49f1167e51f5fde514011761c9fd23caa2b2